crates/matrix-sdk-base/src/store/ambiguity_map.rs

```diff
@@ -218,9 +234,8 @@ impl AmbiguityCache {
                 .and_then(|ev| ev.content.displayname.as_deref())
                 .unwrap_or_else(|| member_event.state_key().localpart());
 
-            // We don't allow other users to set the display name, so if we
-            // have a more trusted version of the display
-            // name use that.
+            // We don't allow other users to set the display name, so if we have a more
```

Comment:
(Same question above for `Join | Invite`, do we need to add `| Knock`?)

Reply:
Yes, they need to be the same, refactored things to ensure that the two places where we do this check are in sync: 4532a4db6588cf212ff0108caed36415923728f1.